web_accounts_list.json

```diff
@@ -1748,6 +1759,17 @@
          "category" : "news",
          "valid" : true
       },
+      {
+         "name" : "Tik Tok",
+         "check_uri" : "https://www.tiktok.com/@{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : " Following",
+         "account_missing_string" : "Couldn&#x27;t find this account",

```

Comment:
Can you check this account_missing_string? Tried view-source:https://www.tiktok.com/@jennifer55555 and cannot find the literal string. The string could be found with a regular ```'```. Also, I found that string in a valid user account HTML.

Reply:
Test it using Python instead of a browser. This should check out as proposed. It does in my end. Your browser must be translating the html entity. Remember a user agent - else you'll get json and not the html.